### PR TITLE
build controller ci testing

### PR DESCRIPTION
**Pull request by k-rister**

### CODE REVIEW

File: .github/workflows/crucible-ci.yaml [hunk 1]
```diff
@@ -13,7 +13,35 @@ on:
...
+        diff_cmd="git diff --name-only HEAD^1 HEAD"
+        echo "Files changed:"
+        ${diff_cmd}
+        echo
+        workshop_files_changed=$(${diff_cmd} | grep "^workshop/" | wc -l)
+        echo "workshop_files_changed=${workshop_files_changed}"
+        echo
+        if [ ${workshop_files_changed} -gt 0 ]; then answer="yes"; else answer="no"; fi
+        echo "Setting answer to '${answer}'"
+        echo "::set-output name=answer::${answer}"
```
Comment (rafaelfolco) [inline .github/workflows/crucible-ci.yaml:40]: have you seen this ? https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-including-paths

This seems to do the same as you are doing manually...

**Review by rafaelfolco — COMMENTED** [review]
paths: may save you some logic there

File: .github/workflows/crucible-ci.yaml (hunk 1 above)
Comment (k-rister) [reply .github/workflows/crucible-ci.yaml:40]: I'm already using that in this workflow.  What I was trying to do here was avoid having to create yet another workflow that only had a single parameter difference from this one.  By adding this job and creating an output from it that it used as input to the other jobs I was able to keep a single workflow with all the other logic consolidated.

Comment (rafaelfolco) [reply .github/workflows/crucible-ci.yaml:40]: oh I see, any reason you want one single workflow ?

Comment (k-rister) [reply .github/workflows/crucible-ci.yaml:40]: To avoid replication and having to keep the workflows synchronized.

Comment (rafaelfolco) [reply .github/workflows/crucible-ci.yaml:40]: simplicity, fair enough

**Review by rafaelfolco — APPROVED** [review]

**Pull request merged**